components/match2/commons/match.lua

```diff
@@ -385,6 +385,14 @@ end
 function Match._prepareGameRecordForStore(matchRecord, gameRecord)
 	gameRecord.parent = matchRecord.parent
 	gameRecord.tournament = matchRecord.tournament
+	if not gameRecord.participants and gameRecord.opponents then
+		gameRecord.participants = {}
+		for opponentId, opponent in ipairs(gameRecord.opponents) do
+			for playerId, player in ipairs(opponent.players) do
+				gameRecord.participants[opponentId .. '_' .. playerId] = player
+			end
```

Comment:
hmmm that assumes that game.opponent.players is an array without gaps, which to my understanding isn't always true
or (potentially) it attaches the players wrongly again

Reply:
Well depends on how opponents.players is built up ;)
But alright, I'll change it to `pairs` for now